Demo/Application/Base/BaseViewController.swift

```diff
@@ -5,6 +5,7 @@ class BaseViewController: UIViewController {
 
     var progressBlock: ((String?) -> Void) = { _ in }
     var completionBlock: ((BTPaymentMethodNonce?) -> Void) = { _ in }
+    var paypalEditCompletionBlock: ((String) -> Void) = { _ in  }
```

Comment:
I don't think we need this, we can just use the `progressBlock`

Reply:
But it's really a completion Block, not progressBlock, which is used to show progress. 
I thought it might be confusing.

Reply:
The progress block is also used for errors, which are part of the completion. It looks like `completionBlock` here represents anything that's a `BTPaymentMethodNonce` and `progressBlock` is used for anything that's a string, so I think reusing that is fine.